.template-lintrc.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     extends: 'recommended',
 
     rules: {
-        'block-indentation': 4
+        'block-indentation': 4,
+        'bare-strings': false
```

Comment:
Temporary--the console warnings dictated that it at least be there, but since if it were set to true, it would cause test failures (I tried it and there were *a lot*). I will add a ticket to the backlog to fix this. I agree with you, we should have translation support.